src/DomainWebformService.php

```diff
@@ -57,18 +56,27 @@ public function __construct(
   }
 
   /**
-   * Provides the alowed domain of the user.
+   * Provides the allowed domain of the user.
    *
    * @return array
    *   Returns the array of user allowed domains.
    */
   public function getUserAllowedDomains() {
+    $domain_access = $domain_admin = [];
     /** @var /Drupal\user\UserInterface */
     $user = $this->entityTypeManager->getStorage('user')->load($this->currentUser->id());
-    $domain_access = $user->get('field_domain_access')->getValue();
-    $domain_access = array_column($domain_access, 'target_id');
-    $domain_admin = $user->get('field_domain_admin')->getValue();
-    $domain_admin = array_column($domain_admin, 'target_id');
+
+    if ($user->hasField('field_domain_access') &&
+      $domain_access = $user->get('field_domain_access')->getValue()
```

Comment:
This field is created by domain_access (submodule of domain module). If we really need this field or any other features from domain_access module we need to add domain_access as dependency in this module (and remove domain module).